source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/stompy/rough_env_cfg.py

```diff
@@ -0,0 +1,185 @@
+            "sensor_cfg": SceneEntityCfg(
+                "contact_forces",
+                body_names=".*_foot_1_rmd_x4_24_mock_1_outer_rmd_x4_24_1",
+            ),
+            "threshold": 0.4,
+        },
+    )
+    feet_slide = RewTerm(
+        func=mdp.feet_slide,
+        weight=-0.25,
+        params={
+            "sensor_cfg": SceneEntityCfg(
+                "contact_forces",
+                body_names=".*_foot_1_rmd_x4_24_mock_1_outer_rmd_x4_24_1",
+            ),
+            "asset_cfg": SceneEntityCfg(
+                "robot", body_names=".*_foot_1_rmd_x4_24_mock_1_outer_rmd_x4_24_1"
+            ),
+        },
+    )
+    # Penalize ankle joint limits
+    dof_pos_limits = RewTerm(
+        func=mdp.joint_pos_limits,
+        weight=-1.0,
+        params={"asset_cfg": SceneEntityCfg("robot", joint_names=".*ankle.*")},
+    )
+    # Penalize deviation from default of the joints that are not essential for locomotion
+    joint_deviation_hip = RewTerm(
+        func=mdp.joint_deviation_l1,
+        weight=-0.2,
+        params={
+            "asset_cfg": SceneEntityCfg(
+                "robot", joint_names=[".*hip_yaw", ".*hip_roll"]
+            )
+        },
+    )
+    joint_deviation_arms = RewTerm(
+        func=mdp.joint_deviation_l1,
+        weight=-0.2,
+        params={
+            "asset_cfg": SceneEntityCfg(
+                "robot", joint_names=[".*shoulder.*", ".*elbow.*"]
+            )
+        },
+    )
+    joint_deviation_torso = RewTerm(
+        func=mdp.joint_deviation_l1,
+        weight=-0.1,
+        params={"asset_cfg": SceneEntityCfg("robot", joint_names="torso_roll")},
+    )
+
+
+@configclass
+class TerminationsCfg:
+    """Termination terms for the MDP."""
+
+    time_out = DoneTerm(func=mdp.time_out, time_out=True)
+    base_contact = DoneTerm(
+        func=mdp.illegal_contact,
+        params={
+            "sensor_cfg": SceneEntityCfg(
+                "contact_forces", body_names="*.bottom_skeleton_1.*"
+            ),
+            "threshold": 1.0,
+        },
+    )
+
+
+@configclass
+class StompyRoughEnvCfg(LocomotionVelocityRoughEnvCfg):
+    rewards: StompyRewards = StompyRewards()
+    terminations: TerminationsCfg = TerminationsCfg()
+
+    def __post_init__(self):
+        # post init of parent
+        super().__post_init__()
+        # Scene
+        self.scene.robot = STOMPY_CFG.replace(prim_path="{ENV_REGEX_NS}/Robot")
+        self.scene.height_scanner.prim_path = "{ENV_REGEX_NS}/Robot/link_upper_limb_assembly_7_dof_1_torso_1_bottom_skeleton_1"
+
+        # Randomization
+        # TODO: LOOK FOR HYPERPARAMETERS
+        self.events.push_robot = None
+        self.events.add_base_mass = None
+        self.events.reset_robot_joints.params["position_range"] = (1.0, 1.0)
+        self.events.base_external_force_torque.params["asset_cfg"].body_names = [
+            ".*link_upper_limb_assembly_7_dof_1_torso_1_bottom_skeleton_1"
+        ]
+        self.events.reset_base.params = {
```

Comment:
Are these default values?

Reply:
yep they were using a torso link so i matched it as best i could